charts/kong/templates/deployment.yaml

```diff
@@ -6,6 +6,11 @@ metadata:
     {{- include "kong.metaLabels" . | nindent 4 }}
     app.kubernetes.io/component: app
 spec:
+  template:
+    metadata:
+      annotations:
+        kuma.io/gateway: enabled
+        traffic.sidecar.istio.io/includeInboundPorts: ""
```

Comment:
I don't this is what we want. These annotations will be injected into the pods that belong the Deployment but not to the deployment itself.

To add the annotations to the deployment itself, the annotations block should go into the metadata block at line 3 for this file.